lib/legacy/polymer.dom.html

```diff
@@ -349,6 +331,119 @@
    */
   Polymer.DomApi.prototype.querySelectorAll;
 
+  if (window.ShadyDOM && window.ShadyDOM.inUse && window.ShadyDOM.noPatch) {
```

Comment:
Factor this stuff to reuse code above by adding wrappers and using a mixin or copying properties.